lib/xcodeproj/project/object/native_target.rb

```diff
@@ -377,6 +377,27 @@ def add_system_library(names)
         end
         alias_method :add_system_libraries, :add_system_library
 
+        # Adds a file reference for one or more system libraries to the project
```

Comment:
Does this doc need to be updated? 

Also can we extract common logic between this method and `add_system_library` method?